<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Icons/login.png</normaloff>../Icons/login.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(143, 143, 143);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>300</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 255, 127);
</string>
    </property>
    <property name="text">
     <string>Sign up</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Msg">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>410</y>
      <width>240</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="NotEqual_error">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>270</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>not equal</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>302</width>
      <height>217</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="FirstName">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{ 
                background-color: #c6c6c6;
                border: 2px solid gray;
                border-radius: 4px;
                padding: 0 8px;
                selection-background-color: darkgray;
                font-size: 14px;
            }</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LastName">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{ 
                background-color: #c6c6c6;
                border: 2px solid gray;
                border-radius: 4px;
                padding: 0 8px;
                selection-background-color: darkgray;
                font-size: 14px;
            }
</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="StudentNo_LineEdit">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{ 
                background-color: #c6c6c6;
                border: 2px solid gray;
                border-radius: 4px;
                padding: 0 8px;
                selection-background-color: darkgray;
                font-size: 14px;
            }</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="PassLineEdit">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{ 
                background-color: #c6c6c6;
                border: 2px solid gray;
                border-radius: 4px;
                padding: 0 8px;
                selection-background-color: darkgray;
                font-size: 14px;
            }</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Pass_repeat">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{ 
                background-color: #c6c6c6;
                border: 2px solid gray;
                border-radius: 4px;
                padding: 0 8px;
                selection-background-color: darkgray;
                font-size: 14px;
            }</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Email_LineEdit">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{ 
                background-color: #c6c6c6;
                border: 2px solid gray;
                border-radius: 4px;
                padding: 0 8px;
                selection-background-color: darkgray;
                font-size: 14px;
            }</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>350</y>
      <width>195</width>
      <height>30</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="submit_Button">
       <property name="styleSheet">
        <string notr="true">QPushButton::hover
{
background-color: rgb(0, 170, 0);
}</string>
       </property>
       <property name="text">
        <string>submit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clear_Button">
       <property name="styleSheet">
        <string notr="true">QPushButton::hover
{
background-color: rgb(255, 0, 0);
}</string>
       </property>
       <property name="text">
        <string>clear all</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
